sound/hybrid_font.h

```diff
@@ -367,11 +372,16 @@ class HybridFont : public SaberBase {
   bool swinging_ = false;
   void SB_Motion(const Vec3& gyro, bool clear) override {
     float speed = sqrtf(gyro.z * gyro.z + gyro.y * gyro.y);
+    if (millis() - last_swing_millis_ >= 10 ) {
```

Comment:
This is going to be a bit uneven.
I would think that the acceleration is changing relatively sharply around the time that the swing triggers.
So it will then depend on where in the 10-millisecond cycle the swing occurs.

Maybe something like this would work better:
   uint32_t now = micros();
   uint32_t delta_micros = now - last_swing_micros_;
   last_swing_micros_ = now;
   float delta = delta_micros * 0.000001;
   float current_acceleration  = (speed - last_speed_) / delta;
   float filter_factor = powf(0.2, delta);
   swing_acceleration_ = swing_acceleration_ * filter_factor + current_acceleration_ * (1 - filter_factor);

Ultimately, it might be that acceleration isn't the best way to do this though.

Reply:
Provided I reference this value as an absolute, it works great.  If I don't, the values are mostly negative.  Changing the filter factor doesn't make any difference.